README.md

```diff
@@ -167,6 +167,7 @@ Make sure that whatever you speicfy for the new hostname is compliant with the r
     chmod +x setup-teslausb
     ./setup-teslausb
     ```
+    > Note: If the setup-teslausb command exits very quickly with only "adding country code to wpa_supplicant.conf", you can just run ```./setup-teslausb``` again.
```

Comment:
wouldn't it also have printed "please reboot for changes to take effect" in that case?

Reply:
Yes, it did, and this whole path led to a botched install (never connected to WiFi, and started showing "USB device malfunctioned" on Windows 10, so I am going to retract this until I figure out what's actually going on, work through everything manually, and make a better suggestion next time. :-)